pkg/sync/destination.go

```diff
@@ -76,6 +92,11 @@ func NewImageDestination(registry, repository, tag, username, password string, i
 	}, nil
 }
 
+// isPermanentServiceAccountToken returns true if user is a Google permanent service account token
+func isPermanentServiceAccountToken(registry string, username string) bool {
```

Comment:
Maybe this function is better to be put into pkg/sync/utils.go.

Reply:
Have moved the function